src/json.c

```diff
@@ -33,29 +33,29 @@ along with GNU Emacs.  If not, see <https://www.gnu.org/licenses/>.  */
 #define JSON_HAS_ERROR_CODE (JANSSON_VERSION_HEX >= 0x020B00)
 
 #ifdef WINDOWSNT
-# include <windows.h>
-# include "w32common.h"
-# include "w32.h"
+#include <windows.h>
+#include "w32.h"
+#include "w32common.h"
```

Comment:
What is the purpose of this change? If it's really necessary, why hasn't it been made upstream?

Reply:
I guess clangd has performed that automatically(the changes to this file will be reverted once lisp_fn is in).